Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+#!groovy
+
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+  }
+
+  stages {
+    stage('Build Docker image') {
+      agent any
+      steps {
+        script {
+          def nginxVersion = '1.19'
```

Comment:
So we opt-in to version upgrades by manually updating this?

I'm wondering how we will remember to update these images to release to the next version...

Reply:
Yeah I'm not sure how we'll remember that either tbh. We'll need to keep this updated and then also keep the various deployment templates for apps updated.

What we really need is some way to hook that all up to Dependabot or something.